server/services/signup/hooks/attachPosition.js

```diff
@@ -38,7 +38,7 @@ module.exports = () => (hook) => {
       return models.event.findOne(query)
         .then((event) => {
           const currentQuota = _.find(event.quota, { dataValues: { id: quotaId } }).dataValues;
-          const positionInQuota = currentQuota.signupsBefore + 1;
+          const positionInQuota = currentQuota.signupsBefore;
```

Comment:
Maybe the attribute in the model should be renamed or the logic changed, as the value is actually the current available place and not the last filled one?